lib/Doctrine/ORM/Tools/Console/Command/ConvertDoctrine1SchemaCommand.php

```diff
@@ -180,6 +180,7 @@ public function convertDoctrine1Schema(array $fromPaths, $destPath, $toType, $nu
                 );
             }
         }
+        unset($dirName);
```

Comment:
Not sure if I like this inspection. `$dirName` is not used after this location: let the stack garbage collection do its job.